saleor/order/actions.py

```diff
@@ -766,6 +766,7 @@ def automatically_fulfill_digital_lines(
         send_fulfillment_confirmation_to_customer(
             order, fulfillment, user=order.user, app=None, manager=manager
         )
+        manager.fulfillment_created(fulfillment)
```

Comment:
There is `get_or_create` in line 733 so there might be case where the fulfillment is already created and the notification was sent before?

Reply:
in this implementation yes as notify event. Do we want to change the logic and send notification for any new fulfillment?